utils/report.c

```diff
@@ -769,6 +776,14 @@ void report_sort_tasks(struct uftrace_data *handle, struct rb_root *name_root,
 	}                                                                                          \
 	FIELD_STRUCT(_id, _name, _func, _header, 10)
 
+#define FIELD_TIMESTAMP(_id, _name, _field, _func, _header)                                        \
+	static void print_##_func(struct field_data *fd)                                           \
+	{                                                                                          \
+		struct uftrace_report_node *node = fd->arg;                                        \
+		print_time_unit_padding(node->_field, 2);                                          \
```

Comment:
Nop, timestamp should not be printed using `print_time_unit()`.  Please add something similar to `print_timestamp()` in cmds/replay.c file.

Reply:
I think I have fixed these two problems.